### PR TITLE
add 110 scam urls

### PR DESCRIPTION
**Pull request by blocksecscamreport**
Hello, we are from blocksec(https://blocksec.com/), recently our system detects many phishing domains(https://github.com/blocksecteam/web3_phishing_domains) and extracts 110 scam urls. The phishing behaviors only include approve, transfer. These domains need to be blocked, details here:
| reported url | phishing domain | phishing_behavior |
| :----: | :----: | :----: |
| ethereumbal.com | https://ethereumbal.com | Approve USDT Token To EOA address 0xb576859a3b150bae6a748d8fc9a82ebfc1389386 |
| ethereum-che.com | https://ethereum-che.com | Approve USDT Token To EOA address 0x2ad0875e63116e96eec7c4d38a726457b628dd68 |
| ethereumche.com | https://ethereumche.com | Approve USDT Token To EOA address 0xce064d7145a30c5aea11e64b77723bccf12b779e |
| ethereum-gmax.com | https://ethereum-gmax.com | Approve USDT Token To EOA address 0x7b92fb979ae44b646121e5ec07abf82a7022883d |
| ethereumgmax.com | https://ethereumgmax.com | Approve USDT Token To EOA address 0xb576859a3b150bae6a748d8fc9a82ebfc1389386 |
| ethereum-srm.com | https://ethereum-srm.com | Approve USDT Token To EOA address 0x2ad0875e63116e96eec7c4d38a726457b628dd68 |
| ethereumsrm.com | https://ethereumsrm.com | Approve USDT Token To EOA address 0xce064d7145a30c5aea11e64b77723bccf12b779e |
| ethereum-uma.com | https://ethereum-uma.com | Approve USDT Token To EOA address 0x2ad0875e63116e96eec7c4d38a726457b628dd68 |
| ethereumuma.com | https://ethereumuma.com | Approve USDT Token To EOA address 0xce064d7145a30c5aea11e64b77723bccf12b779e |
| ethereum-woo.com | https://ethereum-woo.com | Approve USDT Token To EOA address 0x7b92fb979ae44b646121e5ec07abf82a7022883d |
| usdtalgorand.com | https://usdtalgorand.com | Approve USDT Token To EOA address 0x7d60695116f5608626e8f95d6de923a7db4bf634 |
| usdtcompound.com | https://usdtcompound.com | Approve USDT Token To EOA address 0x7d60695116f5608626e8f95d6de923a7db4bf634 |
| usdtloopring.com | https://usdtloopring.com | Approve USDT Token To EOA address 0x7d60695116f5608626e8f95d6de923a7db4bf634 |
| whitelist-zkairdrop.com | https://whitelist-zkairdrop.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| whitelist-zkbridge.com | https://whitelist-zkbridge.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| whitelist-zknetwork.com | https://whitelist-zknetwork.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| zkairdrop-whitelist.com | https://zkairdrop-whitelist.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| zkbridge-whitelist.com | https://zkbridge-whitelist.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| zkcsyncs.com | https://zkcsyncs.com | Approve USDT Token To EOA address 0x5a8fd953eff277a3b98fd6e6009ec63b8e5a97e6 |
| zkcysync.com | https://zkcysync.com | Transfer USDT Token To EOA address 0x977628bdae27d8f6f2ef67429cdbdeddd06f6670 |
| zknetwork-whitelist.com | https://zknetwork-whitelist.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| zkskyn.com | https://zkskyn.com | Approve USDT Token To EOA address 0x5a8fd953eff277a3b98fd6e6009ec63b8e5a97e6 |
| zksync.drops.su | https://zksync.drops.su | Transfer USDT Token To EOA address 0x9d8a3f49888d23594a2efcd2f3c5fa8145541a51 |
| zk-sync.re | https://zk-sync.re | Approve USDT Token To EOA address 0xd13797007bae65320c78f4ae92f5f595769d441f |
| zksyncswap.xyz | https://zksyncswap.xyz | Transfer USDT Token To EOA address 0x704e942ce766ab7aa472ac3460787c808e80da72 |
| aribitrum-drop.su | https://aribitrum-drop.su | Transfer USDT Token To EOA address 0x8cccdc0da2d3c6ad4a754db4812b9eef19877eec |
| claims-wbt.com | https://claims-wbt.com | Approve USDT Token To EOA address 0x908e4b083d55485bade5e0c4a69e8c3055fadb52 |
| claim.zetachain.vip | https://claim.zetachain.vip | Approve USDT Token To EOA address 0xff30b32c7e7da16cc7cd100a54ecd77b103d1a1c |
| claim.zetachajn.com | https://claim.zetachajn.com | Approve USDT Token To EOA address 0xff30b32c7e7da16cc7cd100a54ecd77b103d1a1c |
| cryptogpt.page | https://cryptogpt.page | Transfer ETH To EOA address 0xadbc232e01cd3dd51e013b0c90475dd4602d2638 |
| dogecoin-musk.com | https://dogecoin-musk.com | Approve USDT Token To EOA address 0x8db2be9e33fd9407542c88eae4603351ecbb201b |
| dogegot.com | https://dogegot.com | Approve USDT Token To EOA address 0x8db2be9e33fd9407542c88eae4603351ecbb201b |
| ethereum-lp.com | https://ethereum-lp.com | Approve USDT Token To EOA address 0x2724e57b2bad79024dee4233c5aed3280dbd83ad |
| ethrandom.net | https://ethrandom.net | Approve USDT Token To EOA address 0x1cf5dfb045a068df4b105d58f195e73e8b115665 |
| filecoin.red | https://filecoin.red | Approve USDT Token To EOA address 0x1cd5af1232f537d5eb9dffb6fc7242f871ce9a2b |
| kandyland.land | https://kandyland.land | Approve USDT Token To EOA address 0xd28b8d7d6cde59b3d8314c9b055ade963331f7bf |
| kingdatasa.co | https://kingdatasa.co | Approve USDT Token To EOA address 0x7063b153d15ef9303c62284bc2405aaace322492 |
| kingdatasa.com | https://kingdatasa.com | Approve USDT Token To EOA address 0x7063b153d15ef9303c62284bc2405aaace322492 |
| kodapandent-otherside.com | https://kodapandent-otherside.com | Approve USDT Token To EOA address 0xd5dea80bef02942c9787434b804b7202697623a4 |
| lovehateinu.verification-kyc.com | https://lovehateinu.verification-kyc.com | Approve USDT Token To EOA address 0x29c438d9f0e3cdea2e8b616b3726dd63dec04a51 |
| mint.killabulls.com | https://mint.killabulls.com | Approve USDT Token To EOA address 0x38a18307fdfe3f2ad5377b674ab65b539d9d6461 |
| mint.robogenesis.xyz | https://mint.robogenesis.xyz | Approve USDT Token To EOA address 0x38a18307fdfe3f2ad5377b674ab65b539d9d6461 |
| myria-airdrop.com | https://myria-airdrop.com | Approve USDT Token To EOA address 0x49dc14dd851b6eae8d685715e12a06cc1bfc5d8d |
| nakamigos.co | https://nakamigos.co | Approve USDT Token To EOA address 0x276c094b89367eefad5f3a8545e692cf8a7512e8 |
| nftotherside.net | https://nftotherside.net | Approve USDT Token To EOA address 0xff30b32c7e7da16cc7cd100a54ecd77b103d1a1c |
| otherside.pm | https://otherside.pm | Approve USDT Token To EOA address 0xd13797007bae65320c78f4ae92f5f595769d441f |
| raiffeisendecred.com | https://raiffeisendecred.com | Approve USDT Token To EOA address 0x3951d8186bf618c69228c85c7ba5b5e972d8d448 |
| raiffeisengolem.com | https://raiffeisengolem.com | Approve USDT Token To EOA address 0x3951d8186bf618c69228c85c7ba5b5e972d8d448 |
| seimetwork.com | https://seimetwork.com | Transfer USDT Token To EOA address 0xa39fd8ebe1952283ba876da6dc46182f0f047034 |
| shibrariumtech.com | https://shibrariumtech.com | Approve USDT Token To EOA address 0x81312d3e749336053f74eced8d29670b20d13a73 |
| sui-community.com | https://sui-community.com | Approve USDT Token To EOA address 0x893a3e8de58a2eae9b1cd304e00fe7e0b62ad7af |
| swapdashboard.info | https://swapdashboard.info | Transfer USDT Token To EOA address 0xcd3cb83f1e66ce11d84c0da516cdbe5e20c72711 |
| thegraphid.club | https://thegraphid.club | Transfer ETH To EOA address 0xf7605f033a8b246bfc90e9f3c70c21a9537986b5 |
| thegraphid.pro | https://thegraphid.pro | Transfer ETH To EOA address 0xf7605f033a8b246bfc90e9f3c70c21a9537986b5 |
| thegraphid.us | https://thegraphid.us | Transfer ETH To EOA address 0xf7605f033a8b246bfc90e9f3c70c21a9537986b5 |
| theshoogans.com | https://theshoogans.com | Approve USDT Token To EOA address 0x6f935c7b20a61ff52013f02170f3868b7913d05c |
| 2023trustpad.top | https://2023trustpad.top | Transfer ETH To EOA address 0x6aca7aa6dd77f6b3f2845376347c380f4286461b |
| airdrop-500.top | https://airdrop-500.top | Transfer ETH To EOA address 0x6fa039551c6102b407d6fffd198f0d85cba049dc |
| airdrop-cryptogpt.com | https://airdrop-cryptogpt.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| anapecalledbeast.com | https://anapecalledbeast.com | Transfer ETH To EOA address 0x9a87d4c0a898f3c4c0ed47c551ee10adf234f8b4 |
| arb-nft.com | https://arb-nft.com | Transfer ETH To EOA address 0x2ca1148f6afc30ae57f045781b38320861cd9482 |
| bdmexhaust.com | https://bdmexhaust.com | Approve USDT Token To EOA address 0x1d81c65ef459fd78de30633191d9f858f6bdff04 |
| blvckparis-nft.pages.dev | https://blvckparis-nft.pages.dev | Transfer ETH To EOA address 0x10091b4df8f2f54562b32b63dba82ac3ec396be1 |
| claim500usd.top | https://claim500usd.top | Transfer ETH To EOA address 0x6aca7aa6dd77f6b3f2845376347c380f4286461b |
| claim-hilumia.com | https://claim-hilumia.com | Approve USDT Token To EOA address 0xd5dea80bef02942c9787434b804b7202697623a4 |
| cloudbets.xyz | https://cloudbets.xyz | Transfer ETH To EOA address 0xca59a61ca4276c536f22809d9a42759528fad9df |
| connect.walletsyncc.com | https://connect.walletsyncc.com | Transfer USDT Token To EOA address 0xee9260bf473b7d7c3cbaba225e8fbbf075fca58a |
| cryptoamlbot.site | https://cryptoamlbot.site | Approve USDT Token To EOA address 0x63f7aa9a9ceab424d3b4ff81fd39d9b9560a2059 |
| cyberknights.fun | https://cyberknights.fun | Transfer ETH To EOA address 0x38d53b97c35daba7fd87aa0bddc48eea31fac993 |
| ecosystem-arbitrum.com | https://ecosystem-arbitrum.com | Approve USDT Token To EOA address 0x908e4b083d55485bade5e0c4a69e8c3055fadb52 |
| enter-doodles.com | https://enter-doodles.com | Approve USDT Token To EOA address 0x908e4b083d55485bade5e0c4a69e8c3055fadb52 |
| gemesis.pro | https://gemesis.pro | Transfer ETH To EOA address 0xcd66563babb4b331c85d88c5e6a898b0e2fcd8b3 |
| kandylandmint.xyz | https://kandylandmint.xyz | Approve USDT Token To EOA address 0xf12780fe502839a673c79ac98166abab0bbc1f30 |
| kodapandant-otherside.com | https://kodapandant-otherside.com | Approve USDT Token To EOA address 0xd5dea80bef02942c9787434b804b7202697623a4 |
| lovehateinu.one | https://lovehateinu.one | Approve USDT Token To EOA address 0x1cd5af1232f537d5eb9dffb6fc7242f871ce9a2b |
| metacade.one | https://metacade.one | Approve USDT Token To EOA address 0x1cd5af1232f537d5eb9dffb6fc7242f871ce9a2b |
| metawin-nft.tech | https://metawin-nft.tech | Approve USDT Token To EOA address 0x345e3789538cfb1d5b030cfce8ca18532c871aab |
| mg-land.land | https://mg-land.land | Approve USDT Token To EOA address 0xffbaf49bf8580a3adb9ba28f46dae6631e2e2c58 |
| miningpool18.com | https://miningpool18.com | Approve USDT Token To EOA address 0x9ed27728aca1461e84b8f8bb60d7786c6496dff5 |
| mint.na4u.ru | https://mint.na4u.ru | Transfer ETH To EOA address 0x60ab839ba8450fd362044bf2e2de970d013c8379 |
| nft-degods.com | https://nft-degods.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| othersidevessels.com | https://othersidevessels.com | Approve USDT Token To EOA address 0x49dc14dd851b6eae8d685715e12a06cc1bfc5d8d |
| paraverse.one | https://paraverse.one | Approve USDT Token To EOA address 0x1cd5af1232f537d5eb9dffb6fc7242f871ce9a2b |
| raffle-takanotsumedan.com | https://raffle-takanotsumedan.com | Approve USDT Token To EOA address 0x6f935c7b20a61ff52013f02170f3868b7913d05c |
| raiffeisenbinaryx.com | https://raiffeisenbinaryx.com | Approve USDT Token To EOA address 0x3951d8186bf618c69228c85c7ba5b5e972d8d448 |
| raiffeisenlivepeer.com | https://raiffeisenlivepeer.com | Approve USDT Token To EOA address 0x3951d8186bf618c69228c85c7ba5b5e972d8d448 |
| raiffeisensafeoal.com | https://raiffeisensafeoal.com | Approve USDT Token To EOA address 0x3951d8186bf618c69228c85c7ba5b5e972d8d448 |
| rewards-nakamigos.com | https://rewards-nakamigos.com | Approve USDT Token To EOA address 0xd5dea80bef02942c9787434b804b7202697623a4 |
| shipwreck-memeland.com | https://shipwreck-memeland.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| signin-mdvmm.com | https://signin-mdvmm.com | Approve USDT Token To EOA address 0xd5dea80bef02942c9787434b804b7202697623a4 |
| spaceid-claim.space | https://spaceid-claim.space | Transfer USDT Token To EOA address 0x225ca6fdc948b6f4a8d905c2020733c798a78853 |
| thefloki.online | https://thefloki.online | Approve USDT Token To EOA address 0xced5341d789c7eb5eb37e528bd7580f731fb4b0d |
| theyogapetz.com | https://theyogapetz.com | Transfer ETH To EOA address 0xd1f768b12bf10b435deb53e7a88cc01fdbdb91e2 |
| toshies.li-vedrop.com | https://toshies.li-vedrop.com | Approve USDT Token To EOA address 0x1661f1b207629e4f385da89cff535c8e5eb23ee3 |
| victoriavr-token.com | https://victoriavr-token.com | Approve USDT Token To EOA address 0x6317d43b1738a9c0f185ab4cb89901b8b78976d4 |
| web3-lens.art | https://web3-lens.art | Approve USDT Token To EOA address 0x6f935c7b20a61ff52013f02170f3868b7913d05c |
| web3-lens.com | https://web3-lens.com | Approve USDT Token To EOA address 0x6f935c7b20a61ff52013f02170f3868b7913d05c |
| xxdead.org | https://xxdead.org | Approve USDT Token To EOA address 0x00b63e4edebc022a49c1693e4c874cac0bb99318 |
| yu-gi-yn.xyz | https://yu-gi-yn.xyz | Transfer ETH To EOA address 0x215e34c6ee94d3614403092107a3276124aed903 |
| airdrop-trustpads.top | https://airdrop-trustpads.top | Transfer ETH To EOA address 0x6fa039551c6102b407d6fffd198f0d85cba049dc |
| alphadoggg.limited-premint.com | https://alphadoggg.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| arbitrim.com | https://arbitrim.com | Transfer ETH To EOA address 0xadbc232e01cd3dd51e013b0c90475dd4602d2638 |
| avalanchdrop.com | https://avalanchdrop.com | Approve USDT Token To EOA address 0x64c9066cbd741093abba7ae990afc6e671c4abe6 |
| babezdna.limited-premint.com | https://babezdna.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| blur-claim.space | https://blur-claim.space | Transfer USDT Token To EOA address 0x225ca6fdc948b6f4a8d905c2020733c798a78853 |
| cbridge-networsks.com | https://cbridge-networsks.com | Transfer USDT Token To EOA address 0x86abe2cb29dddc47d9f59bafab23ec1162b13a9d |
| cryptoaivision.space | https://cryptoaivision.space | Approve USDT Token To EOA address 0xefd4e3e3322a680b9dae08f1ed06a11001658566 |
| debox.limited-premint.com | https://debox.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| deepblack.limited-premint.com | https://deepblack.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| delabs.imints.xyz | https://delabs.imints.xyz | Approve USDT Token To EOA address 0xd515fbad8d11c0c755a9e029dc3d3d2add5d9a3b |
| doge-project.limited-premint.com | https://doge-project.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| duiduinft.limited-premint.com | https://duiduinft.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| easyzoom.limited-premint.com | https://easyzoom.limited-premint.com | Transfer USDT Token To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| ecproevents.com | https://ecproevents.com | Transfer ETH To EOA address 0x27c6cb2af57f66be3337eee05efe1d6e14017438 |
| ethereum-shanghai.com | https://ethereum-shanghai.com | Transfer USDT Token To EOA address 0xd2738f3cf5541ec4def68c2bd18992a3cc97e2aa |
| evolving-pixels.xyz | https://evolving-pixels.xyz | Approve USDT Token To EOA address 0x63605e53d422c4f1ac0e01390ac59aaf84c44a51 |
| galxe.tokens-drop.com | https://galxe.tokens-drop.com | Approve USDT Token To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| gemesisbyopenseapro.limited-premint.com | https://gemesisbyopenseapro.limited-premint.com | Transfer ETH To EOA address 0xa2b6a7bd0176f137e9978c156593d3597f0a8e40 |
| get-zksync.info | https://get-zksync.info | Approve USDT Token To EOA address 0x49dc14dd851b6eae8d685715e12a06cc1bfc5d8d |

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -515,6 +515,124 @@
     "launchpad.ethereum.org"
   ],
   "blacklist": [
+    "gemesisbyopenseapro.limited-premint.com",
+    "galxe.tokens-drop.com",
+    "evolving-pixels.xyz",
+    "ethereum-shanghai.com",
+    "ecproevents.com",
+    "easyzoom.limited-premint.com",
```
Comment (legobeat) [inline src/config.json:523]: _If_ these subdomains are indeed all in scope, `limited-premint.com` seems warranted its own entry?

Comment (blocksecscamreport) [reply src/config.json:523]: Thanks for your advice, next time we would extract the common parent domain to simplify the PR.

File: src/config.json [hunk 2]
```diff
@@ -515,6 +515,124 @@
...
+    "seimetwork.com",
+    "raiffeisengolem.com",
+    "raiffeisendecred.com",
+    "otherside.pm",
+    "nftotherside.net",
+    "nakamigos.co",
+    "myria-airdrop.com",
+    "mint.robogenesis.xyz",
+    "mint.killabulls.com",
+    "lovehateinu.verification-kyc.com",
```
Comment (legobeat) [inline src/config.json:596]: This one also looks like the TLD could be extracted

File: src/config.json [hunk 3]
```diff
@@ -515,6 +515,124 @@
...
+    "kingdatasa.com",
+    "kingdatasa.co",
+    "kandyland.land",
+    "filecoin.red",
+    "ethrandom.net",
+    "ethereum-lp.com",
+    "dogegot.com",
+    "dogecoin-musk.com",
+    "cryptogpt.page",
+    "claim.zetachajn.com",
```
Comment (legobeat) [inline src/config.json:607]: Extract TLD?

File: src/config.json [hunk 4]
```diff
@@ -515,6 +515,124 @@
...
+    "claim.zetachain.vip",
```
Comment (legobeat) [inline src/config.json:608]: Extract TLD?

File: src/config.json (hunk 1 above)
Comment (legobeat) [reply src/config.json:523]: Great. Why not in this PR?

Comment (blocksecscamreport) [reply src/config.json:523]: We have extract the parent domains that repeatly appears.
As you know, we have an internal system that collects phishing domains, accounts, manually verify the evidence and makes reports.
We would optimize the report system based on your advice.

**Pull request closed without merging**